### DISCUSSION TITLE
Ability to add the admissions webhook

### DISCUSSION THREAD

**Discussion by mal-berbatovci** [prometheus-operator/kube-prometheus#596] [General] [answered]
Hi, 

I'm trying to move from defining the prometheus-operator with helm to jsonnet (Tanka more specifically). So far it's been pretty smooth, but I'm struggling to find an easy way to add the admissions webhook for validating prometheusRules. 

I've managed to find this [commit](https://github.com/coreos/prometheus-operator/pull/3232/commits/30dd810f4b0b437f8c077dcede27004daee7aa33) which updates the docs to explain adding an admission webhook. This seems to be different to how it is added when defining

```yaml
admissionWebHooks: 
  enabled: true
```

As I can't seem to see the additional flags mentioned in the docs on the prometheus operator container when the admissionWebHook is enabled using Helm. 

Is there an easy way to add this using the kube-prometheus jsonnet library?

Thanks

**Comment by brancz** [comment, marked as answer]
I don't think this has been integrated in kube-prometheus yet. I think this is a great addition though! It shouldn't be that complicted, we just need to add the webhook configurations pointing at the prometheus operator.
